kalman_watch/kalman_perigee_mon.py

```diff
@@ -0,0 +1,844 @@
+"""Watch Kalman star data during perigee passages.
+"""
+
+
+import argparse
+import calendar
+import json
+import re
+from itertools import cycle
+from pathlib import Path
+from typing import List, Union
+
+import astropy.units as u
+import numpy as np
+import plotly.express as px
+import plotly.graph_objects as pgo
+from acdc.common import send_mail
+from astropy.table import Table, vstack
+from cheta.fetch import MSIDset
+from cheta.utils import logical_intervals
+from cxotime import CxoTime
+from jinja2 import Template
+from kadi.commands import get_observations
+from kadi.commands.commands_v2 import get_cmds
+from kadi.commands.states import get_states, reduce_states
+from plotly.subplots import make_subplots
+from ska_helpers.logging import basic_logger
+from ska_helpers.run_info import log_run_info
+
+from kalman_watch import __version__
+
+LOGGER = basic_logger(__name__, level="INFO")
```

Comment:
I would not use ska_helpers.logging and just configure in the main function. See sot/ska_helpers/issues/24. I believe the new ska_helpers should not be included in the coming release unless that is fixed, and that would block this.

Reply:
I think that `ska_helpers.logging` is just fine as long as no other modules explicitly configure a root logger with `basicConfig`. But this isn't an issue in `ska_helpers.logging`, it is fundamentally the way that `logging` works.

I suppose here I could do `LOGGER.propagate = False`, but that fixing a problem that doesn't exist here.